### DISCUSSION TITLE
Handle HTML tags inside of metadata

### DISCUSSION THREAD

**Discussion by brockfanning** [open-sdg/open-sdg#1403] [Ideas]
**Is your feature request related to a problem? Please describe.**

As countries turn to using the Word template for maintaining metadata, or pulling metadata from SDMX, there will be HTML tags in the metadata. Actually there has always been the possibility of HTML tags in metadata, but now it will be more common. In particular, there will be H1 and H2 tags.

**Describe the solution that you need**

The Open SDG style needs to be able to handle these tags and still look consistent.

**Why do you need this feature?**

Because it will be more and more common and will be a requested feature.

**Comment by LucyGwilliamAdmin** [comment]
@brockfanning I think this is already possible - see Indicator description and Definitions here: http://uk-sdg-feature-branches.s3-website.eu-west-2.amazonaws.com/feature-html-in-meta/1-1-1/

Or is there more needed?

**Comment by brockfanning** [comment]
@LucyGwilliamAdmin I think this was more of a design ticket - do we need to change the styles at all to make that heading look different? If not, I think we can close this.

**Comment by LucyGwilliamAdmin** [comment]
Ah right, I see - that's fine, I'll add it to the To discuss column

**Comment by LucyGwilliamAdmin** [comment]
@brockfanning can any html heading be used in sdmx word template (i.e. 1-6)?

**Reply by LucyGwilliamAdmin** [reply]
@brockfanning just pinging again
